libcodechecker/libhandlers/parse.py

```diff
@@ -72,6 +73,29 @@ def add_arguments_to_parser(parser):
                         help="Specify the format the analysis results were "
                              "created as.")
 
+    output_opts = parser.add_argument_group("export arguments")
+    output_opts.add_argument('--export',
+                             dest="export",
+                             required=False,
+                             choices=['html'],
+                             help="Specify extra output format type.")
+
+    output_opts.add_argument('-o', '--output',
+                             dest="output_path",
+                             required='--export' in sys.argv,
```

Comment:
This line here is naughty. Users could simply say `--e html` instead of `--export`, because argparse expands shorthand optstrings too.

If you want to set up a mandatory relation between two arguments, a separate `__handle()` method is the best way to go. Examples have been introduced into the codebase, see `server.py` handler for this.